docs/qa/method.md

```diff
@@ -0,0 +1,150 @@
+---
+order: 1
+title: Method
+---
+
+The (first iteration of the) QA process as described [here][releases] is applied to version v0.34.x
+in order to have a set of results acting as benckmarking baseline.
+This baseline is then compared with results obtained in later versions.
+
+Out of the testnet-based test cases described in [releases][releases] we are focusing on two of them:
+_200 Node Test_, and _Rotating Nodes Test_.
+
+[releases]: https://github.com/tendermint/tendermint/blob/v0.37.x/RELEASES.md#large-scale-testnets
+
+# 200 Node Testnet
+
+## Running the test
+
+This section explains how the tests were carried out for reproducibility purposes.
+
+### Infrastructure Requirements
+
+* An account at Digital Ocean (DO), with a high droplet limit (>202)
+* The machine to orchestrate the tests should have the following installed:
+    * A clone of the [testnet repository][testnet-repo]
+        * This repository contains all the scripts mentioned in the reminder of this section
+    * [Digital Ocean CLI][doctl]
+    * [Terraform CLI][Terraform]
+    * [Ansible CLI][Ansible]
+
+### Steps
+
+1. Follow steps 1-4 of the `README.md` at the top of the testnet repository to configure Terraform, and `doctl`.
+2. Copy file `testnet200.toml` onto `testnet200.toml` (do NOT commit this change)
+3. Follow steps 5-10 of the `README.md` to configure and start the 200 node testnet
+    * WARNING: Do NOT forget to run `make terraform-destroy` as soon as you are done with the tests
+4. `ssh` into the `testnet-load-runner` and run script XXXXX
+    * This will take about 40 mins to run
+    * It is running 90-seconds-long experiments in a loop with different loads
```

Comment:
I was thinking to add a comment on verifying whether the blocks are being created and Tendermint is running (as we did when trying to set up the testnet). But I guess this will be apparent from step 5?

Reply:
No, you got a point. Let me add an extra step